isaric/parsers/isaric-rapid.toml

```diff
@@ -305,136 +305,136 @@
 
 [visit.treatment_corticosteroid]
   description = "Corticosteroid"
-  combinedType = "any"
+  combinedType = "firstNonNull"
   fields = [
-    { field = "corticost_cmyn", "ref" = "Y/N/NK", description = "Corticosteroid on the first day of admission?" },
-    { field = "daily_corticost_cmyn", "ref" = "Y/N/NK", description = "Corticosteroid?" },
     { field = "overall_corticost_cmyn", "ref" = "Y/N/NK", description = "Corticosteroid?" },
+    { field = "daily_corticost_cmyn", "ref" = "Y/N/NK", description = "Corticosteroid?" },
+    { field = "corticost_cmyn", "ref" = "Y/N/NK", description = "Corticosteroid on the first day of admission?" },
   ]
 
 [visit.treatment_dexamethasone]
   description = "Dexamethasone used?"
-  combinedType = "any"
+  combinedType = "firstNonNull"
 
   [[visit.treatment_dexamethasone.fields]]
-    field = "corticost_cmtrt___1"
+    field = "overall_corticost_cmtrt___1"
     description = "Dexamethasone used?"
-    values = { 0 = false, 1 = true, if = { corticost_cmyn = 1 } }
+    values = { 0 = false, 1 = true, if = { overall_corticost_cmyn = 1 } }
 
   [[visit.treatment_dexamethasone.fields]]
     field = "daily_corticost_cmtrt___1"
     description = "Dexamethasone used?"
     values = { 0 = false, 1 = true, if = { daily_corticost_cmyn = 1 } }
 
   [[visit.treatment_dexamethasone.fields]]
-    field = "overall_corticost_cmtrt___1"
+    field = "corticost_cmtrt___1"
     description = "Dexamethasone used?"
-    values = { 0 = false, 1 = true, if = { overall_corticost_cmyn = 1 } }
+    values = { 0 = false, 1 = true, if = { corticost_cmyn = 1 } }
 
 [visit.treatment_dialysis]
   description = "Dialysis"
-  combinedType = "any"
+  combinedType = "firstNonNull"
```

Comment:
Making sure that firstNotNull is the appropriate combinedType (here and in other treatment fields)? this would mean treatment_dialysis = false if daily_rrt_prtrt is N, regardless of whether ccm_* field values.

Reply:
Now that's confusing ...

Reply:
The decision to use 'firstNonNull' came out of a conversation with Sadie, so maybe checking with her is the best bet.

But my guess is it avoids potential conflicts? If daily_rrt_prtrt is recorded as Y but cmm* is N (e.g. because they didn't go into ICU and for some reason that portion of the form was filled in, or they didn't receive that treatment while in ICU but did beforehand...), we still want to record only one answer, and the priorities provided control which is used.

Reply:
Sure, makes sense. Will make a ticket to create a mapping guide (could also use the wiki) which will have best practices.